tests/fixtures/metadata.py

```diff
@@ -184,6 +188,67 @@ def item_lib_martigny(
     return item
 
 
+@pytest.fixture(scope="module")
+def item2_lib_martigny_data(data):
+    """Load item of martigny library."""
+    return deepcopy(data.get('item5'))
+
+
+@pytest.fixture(scope="function")
+def item2_lib_martigny_data_tmp(data):
+    """Load item of martigny library scope function."""
+    return deepcopy(data.get('item5'))
+
+
+@pytest.fixture(scope="module")
+def item2_lib_martigny_requested(
+        app,
+        document,
+        item2_lib_martigny_data,
+        loc_public_martigny,
+        item_type_standard_martigny,
+        patron_martigny_no_email,
+        librarian_martigny_no_email,
+        circulation_policies):
+    """Create item of martigny library."""
+    item = Item.create(
+        data=item2_lib_martigny_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(ItemsSearch.Meta.index)
+    transaction_date = datetime.now(timezone.utc).isoformat()
+
+    item.request(
+        patron_pid=patron_martigny_no_email.pid,
+        transaction_location_pid=loc_public_martigny.pid,
+        transaction_user_pid=librarian_martigny_no_email.pid,
+        transaction_date=transaction_date,
+        pickup_location_pid=loc_public_martigny.pid,
+        document_pid=item.replace_refs()['document']['pid']
+    )
+    flush_index(ItemsSearch.Meta.index)
+    flush_index(LoansSearch.Meta.index)
+    flush_index(NotificationsSearch.Meta.index)
+
+    loan = list(item.get_loans_by_item_pid(item_pid=item.pid))[0]
+    item.validate_request(
+        loan_pid=loan.get('loan_pid'),
+        patron_pid=patron_martigny_no_email.pid,
+        transaction_location_pid=loc_public_martigny.pid,
+        transaction_user_pid=librarian_martigny_no_email.pid,
+        transaction_date=transaction_date,
+        pickup_location_pid=loc_public_martigny.pid,
+        document_pid=item.replace_refs()['document']['pid']
+    )
+    flush_index(ItemsSearch.Meta.index)
+    flush_index(LoansSearch.Meta.index)
+    flush_index(NotificationsSearch.Meta.index)
+    loan = list(item.get_loans_by_item_pid(item_pid=item.pid))[0]
+
+    return item, loan
```

Comment:
bad idea. You can add an extra loan fixture that depends to item.

Reply:
fixed